app/move/controllers/update/document.js

```diff
@@ -50,8 +50,10 @@ class UpdateDocumentUploadController extends UpdateBase {
     }
 
     try {
-      await moveService.update({
-        id: req.getMoveId(),
+      const profile = req.getMove().profile
+
+      await profileService.update({
+        ...profile,
```

Comment:
Do you need to send the whole profile, or can you just send the ID like before? Could prevent updating something you don't want to at this stage.

Reply:
We also have to send the person id too. But dumping any props that aren't those now.